.github/workflows/test.yaml

```diff
@@ -0,0 +1,38 @@
+name: 'Stale issues and PR'
+on:
+  workflow_dispatch:
+    inputs:
+      daysBeforeStale:
+        description: 'days-before-stale'
+        required: false
+        default: 1
+      daysBeforeClose:
+        description: 'days-before-close'
+        required: false
+        default: 1
+      daysBeforeAssigneeReminder:
+        description: 'days-before-assignee-reminder'
+        required: false
+        default: 0
+      daysBeforeReplyReminder:
+        description: 'days-before-reply-reminder'
+        required: false
+        default: 0
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: harupy/stale@mlflow-stale-bot
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 1 days with no activity. Remove stale label or comment or this will be closed in 1 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 1 days with no activity.'
+          days-before-stale: ${{ inputs.daysBeforeStale }}
+          days-before-close: ${{ inputs.daysBeforeClose }}
+          start-date: '2022-06-16T00:00:00Z'
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
```

Comment:
What do these flags do? (Don't see them used anywhere else in the PR)

Reply:
These flags prevent the bot from staling/closing PRs.

Reply:
Got it - thanks :)